examples.py/Library/DXF Export/SimpleExport.pyde

```diff
@@ -0,0 +1,40 @@
+""" Simple DXF Export
+    by Simon Greenwold.
+    
+    Press the 'R' key to export a DXF file.
+    """
```

Comment:
Please format the docstring per

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings